static/app/views/dashboards/editAccessSelector.tsx

```diff
@@ -236,6 +244,7 @@ function EditAccessSelector({dashboard, onChangeEditAccess}: EditAccessSelectorP
         }
       }}
       menuFooter={dropdownFooterButtons}
+      onSearch={debounce(val => void onSearch(val), DEFAULT_DEBOUNCE_DURATION)}
```

Comment:
Similar pattern used in TeamFilter for Issues: https://github.com/getsentry/sentry/blob/f32b2d2b8c14f525a539f926b894d54f98053b53/static/app/views/alerts/list/rules/teamFilter.tsx#L101